express/src/routes/testing-routes.ts

```diff
@@ -170,4 +170,22 @@ router.get('/account', (req, res) => {
     });
 });
 
+// Testing routes for Change your service name page
+router.get('/change-service-name', (req, res) => {
+    res.render("account/change-service-name.njk", {
+        serviceName: 'My juggling service'
```

Comment:
Can the service name be taken from FB than hard coding a random name.

Reply:
@muthukswamy  These are TESTING routes, the backend does not exist, the purpose of this routes is to visualize the forntend, so it can be tested by the BA, Design, and Content. When the backend is created the service name will be dynamic and coming from database or session. So for this purpose it does not really matter where does the service name come from.